Makefile

```diff
@@ -52,28 +54,24 @@ install: ensure_environment ensure_version
 	@echo "✅ Completed installation!"
 
 release: ensure_environment ensure_version
-	@echo "🤖 Creating kyved releases..."
+	@echo "🤖 Creating kyved releases (using timestamp $(BUILD_TIME))..."
 	@rm -rf release
 	@mkdir -p release
 
-	@GOOS=darwin GOARCH=amd64 go build $(BUILD_FLAGS) ./cmd/kyved
-	@tar -czf release/kyved_$(ENV)_darwin_amd64.tar.gz kyved
-	@sha256sum release/kyved_$(ENV)_darwin_amd64.tar.gz >> release/release_$(ENV)_checksum
-
-	@GOOS=darwin GOARCH=arm64 go build $(BUILD_FLAGS) ./cmd/kyved
-	@tar -czf release/kyved_$(ENV)_darwin_arm64.tar.gz kyved
-	@sha256sum release/kyved_$(ENV)_darwin_arm64.tar.gz >> release/release_$(ENV)_checksum
-
-	@GOOS=linux GOARCH=amd64 go build $(BUILD_FLAGS) ./cmd/kyved
-	@tar -czf release/kyved_$(ENV)_linux_amd64.tar.gz kyved
-	@sha256sum release/kyved_$(ENV)_linux_amd64.tar.gz >> release/release_$(ENV)_checksum
-
-	@GOOS=linux GOARCH=arm64 go build $(BUILD_FLAGS) ./cmd/kyved
-	@tar -czf release/kyved_$(ENV)_linux_arm64.tar.gz kyved
-	@sha256sum release/kyved_$(ENV)_linux_arm64.tar.gz >> release/release_$(ENV)_checksum
-
-	@rm kyved
-	@echo "✅ Completed release creation!"
+	@for b in darwin:amd64 darwin:arm64 linux:amd64 linux:arm64; do \
+		os=$$(echo $$b | cut -d':' -f1); \
+		arch=$$(echo $$b | cut -d':' -f2); \
+		echo "➡️ "$$os" "$$arch""; \
+		CGO_ENABLED=0 GOOS=$$os GOARCH=$$arch go build $(BUILD_FLAGS) -o release/kyved_$(ENV)_"$$os"_"$$arch" ./cmd/kyved; \
+		touch -a -m -t $(BUILD_TIME) release/kyved_$(ENV)_"$$os"_"$$arch"; \
+		sha256sum release/kyved_$(ENV)_"$$os"_"$$arch" >> release/release_$(ENV)_checksum; \
+		tar -cf release/kyved_$(ENV)_"$$os"_"$$arch".tar release/kyved_$(ENV)_"$$os"_"$$arch"; \
```

Comment:
use `tar -cf release/kyved_$(ENV)_"$$os"_"$$arch".tar -C release release/kyved_$(ENV)_"$$os"_"$$arch"; \` here or else the tar archive will have the `release` folder included